packages/ffe-file-upload-react/src/FileItem.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { number, func, shape, object, string as stringType } from 'prop-types';
+
+const FileItem = props => (
+    <li key={props.index}>
+        {// File is loading
+        !props.file.document && !props.file.error && (
+            <div className="ffe-file-upload__info-section__stencil">
+                <div className="ffe-file-upload__info-section__stencil__info">
+                    <div className="ffe-file-upload__info-section__stencil__info__background" />
+                    <div className="ffe-file-upload__info-section__stencil__info__text">
+                        {props.file.name}
+                    </div>
+                </div>
+                <button
```

Comment:
Could you throw a `type="button"` on this so it doesn't trigger a submit if it's inside a `form`?